src/System.Data.OleDb/src/System/Data/ProviderBase/DbConnectionPoolIdentity.cs

```diff
@@ -0,0 +1,111 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace System.Data.ProviderBase {
+
+    using System;
+    using System.Collections;
+    using System.Data.Common;
+    using System.Diagnostics;
+    using System.Globalization;
+    using System.Runtime.CompilerServices;
+    using System.Runtime.InteropServices;
+    using System.Security;
+    using System.Security.Permissions;
+    using System.Security.Principal;
+    using System.Threading;
+    using System.Runtime.Versioning;
+
+    [Serializable] // Serializable so SqlDependencyProcessDispatcher can marshall cross domain to SqlDependency.
```

Comment:
Please remove all [Serializable] attributes from types in this PR. We do not want that attribute without good reason, and they do not need it.